src/helpers/helpers.sh

```diff
@@ -43,6 +43,7 @@ if [ "$(is_git_repository)" == true ]; then
   export INITIAL_BRANCH_NAME=$(get_current_branch_name)
   export INITIAL_DIRECTORY=$(pwd)
   export INITIAL_OPEN_CHANGES=$(has_open_changes)
+  export INITIAL_PREVIOUS_BRANCH_NAME=$(get_previous_branch_name)
```

Comment:
+1 on keeping the naming consistent, and distinguishing the initial previous branch from the current previous branch later.